lib/WarningConfigurationUrl.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace SimpleSAML\Module\perun;
+
+use SimpleSAML\Error\Exception;
+use SimpleSAML\Logger;
+use SimpleSAML\Configuration;
+
+/**
+ * Implementation of WarningConfiguration using json url as the source of warning attributes
+ * @package SimpleSAML\Module\perun
+ * @author Dominik Baránek <0Baranek.dominik0@gmail.com>
+ */
+class WarningConfigurationUrl
+{
+    const CONFIG_FILE_NAME = 'module_perun.php';
+    const WARNING_URL = 'disco.warning.url';
+    const WARNING_IS_ON = 'disco.warning.isOn';
```

Comment:
The same as for file WarningConfigurationFile

Reply:
Done